lib/backdrop.dart

```diff
@@ -309,16 +300,7 @@ class _BackdropScaffoldState extends State<BackdropScaffold>
       onWillPop: () => _willPopCallback(context),
       child: Scaffold(
         key: scaffoldKey,
-        appBar: AppBar(
-          title: widget.title,
-          actions: widget.iconPosition == BackdropIconPosition.action
-              ? <Widget>[BackdropToggleButton()] + widget.actions
-              : widget.actions,
-          elevation: 0.0,
-          leading: widget.iconPosition == BackdropIconPosition.leading
-              ? BackdropToggleButton()
-              : null,
-        ),
+        appBar: widget.appBar,
```

Comment:
```suggestion
        appBar: widget.appBar ?? AppBar(
          title: widget.title,	
          actions: widget.iconPosition == BackdropIconPosition.action	
              ? <Widget>[BackdropToggleButton()] + widget.actions	
              : widget.actions,	
          elevation: 0.0,	
          leading: widget.iconPosition == BackdropIconPosition.leading	
              ? BackdropToggleButton()	
              : null,	
        ),
```

Reply:
There is one problem:
I already removed the class `BackdropIconPosition`, as it is not needed anymore.
Flutter's [AppBar](https://api.flutter.dev/flutter/material/AppBar-class.html) provides the same feature using the properties `leading` and `automaticallyImplyLeading`:
> If the leading widget is omitted, but the AppBar is in a Scaffold with a Drawer, then a button will be inserted to open the drawer. Otherwise, if the nearest Navigator has any previous routes, a BackButton is inserted instead. This behavior can be turned off by setting the automaticallyImplyLeading to false. In that case a null leading widget will result in the middle/title widget stretching to start.

So if we add `title` as deprecated property, probably `BackdropIconPosition` has to be re-added as well and made deprecated?

Reply:
yes need to keep all attributes as it is, but all with depreciation message